frontend/src/composables/useShootContext/index.js

```diff
@@ -124,10 +133,29 @@ export function createShootContextComposable (options = {}) {
     return normalizeManifest(object)
   })
 
+  function setExtension (shootManifest, type, extension) {
+    if (!Array.isArray(shootManifest?.spec?.extensions)) {
+      set(shootManifest, 'spec.extensions', [])
+    }
+    const extensions = shootManifest.spec.extensions
+    const index = findIndex(extensions, { type })
```

Comment:
```suggestion
    const index = findIndex(extensions, ['type', type])
```
Why do we always mix usages of propery shorthands? The object shorthand results in https://lodash.com/docs/4.17.15#matches while the property shorthand results in https://lodash.com/docs/4.17.15#matchesProperty. The second has a much better performance and is from my personal point of view much better readable.

Reply:
changed. I prefer the object shorthand. But for consistency it makes sense to use the property shorthand.